ops/ccs-ops-misc/load_test/common/stats.py

```diff
@@ -0,0 +1,274 @@
+"""
+Much of this file is adapted from equivalent Locust code, particularly locust.stats.StatsCSV
```

Comment:
Maybe mention the primary purpose of the class here.

Reply:
There's documentation for the classes within this file already, so I'll expand upon this comment and explain what code should live in `stats.py`.